code/game/machinery/machinery.dm

```diff
@@ -539,13 +539,13 @@ Class Procs:
 
 	if(check_records || check_arrest)
 		var/perpname = perp.get_visible_name(TRUE)
-
 		var/datum/data/record/R = find_security_record("name", perpname)
-		if(check_records && !R)
-			threatcount += 4
-
-		if(check_arrest && R && (R.fields["criminal"] == "*Arrest*"))
-			threatcount += 4
+		if(R && R.fields["criminal"])
+			switch(R.fields["criminal"])
+				if("*Execute*")
+					threatcount += 7
+				if("*Arrest*")
+					threatcount += 5

```

Comment:
Threatcount is 5 instead of 4. Prob unintended

Reply:
Actually intended, as in the beepsky code at /living/human.dm arrest is for 5, so that tweaking this file won't change beepsky behaviour, but turrets.